### DISCUSSION TITLE
FormMegalayout 里面的 object 表单无法进行校验

### DISCUSSION THREAD

**Discussion by zFitness** [alibaba/formily#1959] [Q&A 1.x]
把 object 的表单项放在 FormMegalayout 的组件里面无法进行校验

复现地址： https://codesandbox.io/s/hardcore-hill-3mux3?file=/index.js

**Comment by janryWang** [comment]
1.x的设计问题，对象节点做了特殊问题，建议早日升级2.x，然后针对这个问题的话，能绕过就绕过吧

**Reply by zFitness** [reply]
现在要升级需要重写许多代码，想问问 formily1.x 和 formily2.x 可以共存吗

**Reply by zFitness** [reply]
而且我在项目里面使用到了 alist 这个开源项目，

**Reply by janryWang** [reply]
> 现在要升级需要重写许多代码，想问问 formily1.x 和 formily2.x 可以共存吗

一个工程里是没法共存的，可以考虑用微前端
